package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Feb 12 13:13:35 UTC 2018 - ancor@suse.com
+
+- More reasonable location in the disk for the partitions proposed
+  to make the system bootable (bsc#1073680 and bsc#1076851).
+
```

Comment:
Increase version?

Reply:
Well, I wanted to coordinate with other changes being probably merged today